src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countOfStart = 0;
+
+  return function(a) {
+    return function(b) {
+      if (countOfStart < 3) {
+        countOfStart++;
+
+        return a + b;
+      } else {
+        countOfStart--;
+
+        return `Bzzz... Error!`;
```

Comment:
```suggestion
      countOfStart++
  
      if (countOfStart > 3 & countOfStart % 2 === 0) {
        return 'Bzzz... Error!';
      }

      return a + b;
```

We can simplify it a bit